Settings

DanielEsquivelReynoso opened the pull request:
checking for merge issues, open to suggestions for changes

DanielEsquivelReynoso committed:
build add friend, sign out and setting

DanielEsquivelReynoso committed:
mostly done with styling

DanielEsquivelReynoso committed:
Completed styling for Settings page

DanielEsquivelReynoso committed:
major changes

DanielEsquivelReynoso committed:
reshuffled files to be in components folder

jakemgilfix commented on components/BackButton.js:
```diff
@@ -0,0 +1,21 @@
+    <div>
+      <main>
+        <div className="flex justify-left items-center">
+          <button className="bg-limed-spruce text-white w-[30px] h-[30px] rounded-[30px] ml-[25px] mt-[40px]"><svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 16 24"><path fill="#ffffff" d="M16.67 0l2.83 2.829-9.339 9.175 9.339 9.167-2.83 2.829-12.17-11.996z"/></svg>
```
We should not have inline SVGs. SVGs should be accessible in the /public directory and imported.

jakemgilfix commented on components/BackButton.js:
```diff
@@ -0,0 +1,21 @@
+console.log('BackButton being rendered')
+  return (
+    <div>
+      <main>
```
\<main\> is a container for the main section of the entire app. This should not be used here.

jakemgilfix commented on components/Preference.js:
```diff
@@ -0,0 +1,24 @@
+import Head from 'next/head';
+
+const Preference = (props) => {
+console.log('Preference being rendered')
```
Any console.log() statements should be removed prior to PRs

jakemgilfix commented on components/Preference.js:
```diff
@@ -0,0 +1,24 @@
+const pref = props.preference
+  return (
+    <div>
+      <main>
```
\<main\> shouldn't be used here. Each page should have a maximum of 1 \<main\> tag.

jakemgilfix commented on components/Preferences.js:
```diff
@@ -0,0 +1,44 @@
+import Preference from './Preference';
+
+const Preferences = () => {
+  console.log('Preferences being rendered')
```
Please remove console.log()s before PRs.

jakemgilfix commented on components/ProfilePhoto.js:
```diff
@@ -0,0 +1,21 @@
+import Head from 'next/head';
+
+const ProfilePhoto = () => {
+console.log('ProfilePhoto being rendered')
```
Please remove console.log()s prior to PRs.

jakemgilfix commented on components/ProfilePhoto.js:
```diff
@@ -0,0 +1,21 @@
+console.log('ProfilePhoto being rendered')
+  return (
+    <div>
+      <main>
```
\<main\> should not be used here

jakemgilfix commented on components/ProfilePhoto.js:
```diff
@@ -0,0 +1,21 @@
+const ProfilePhoto = () => {
+console.log('ProfilePhoto being rendered')
+  return (
+    <div>
```
This outer div adds an extra node to the DOM tree but doesn't have any function

jakemgilfix commented on components/SaveChanges.js:
```diff
@@ -0,0 +1,17 @@
+import Head from 'next/head';
+
+const SaveChanges = () => {
+  console.log('SaveChanges being rendered')
```
Please remove console.log()s prior to PRs.

jakemgilfix commented on components/SaveChanges.js:
```diff
@@ -0,0 +1,17 @@
+const SaveChanges = () => {
+  console.log('SaveChanges being rendered')
+  return (
+    <div>
```
Outer div has no function.

jakemgilfix commented on components/SaveChanges.js:
```diff
@@ -0,0 +1,17 @@
+  console.log('SaveChanges being rendered')
+  return (
+    <div>
+      <main>
```
\<main\> should not be used here

jakemgilfix commented on components/SignOut.js:
```diff
@@ -0,0 +1,23 @@
+import Head from 'next/head';
+
+const SignOut = () => {
+  console.log('SignOut being rendered')
```
Please remove console.log()s before PRs.

jakemgilfix commented on components/SignOut.js:
```diff
@@ -0,0 +1,23 @@
+  console.log('SignOut being rendered')
+  return (
+    <div>
+      <Head>
```
\<Head\> is used to insert code into the \<head\> of the page being rendered. It should not be used inside of a component.

jakemgilfix commented on components/SignOut.js:
```diff
@@ -0,0 +1,23 @@
+        <meta name="description" content="Genereted by create next app"/>
+        <link rel="icon" href="/images/favicon.ico" />
+      </Head>
+      <main>
```
\<main\> should not be used inside of a component.

jakemgilfix commented on components/SignOut.js:
```diff
@@ -0,0 +1,23 @@
+      <main>
+
+      <div className="flex justify-center items-end">
+        <button className="text-white bg-limed-spruce px-10 w-[87.7%] h-12 rounded-[14px] mt-3">Sign Out</button>
```
Consider using re-usable button components that will be shared with other parts of the app.

jakemgilfix commented on components/BackButton.js:
```diff
@@ -0,0 +1,21 @@
+const BackButton = () => {
+console.log('BackButton being rendered')
+  return (
+    <div>
```
This outer div only complicates the DOM tree since it does nothing but contain another div.

jakemgilfix commented on components/Preference.js:
```diff
@@ -0,0 +1,24 @@
+console.log('Preference being rendered')
+const pref = props.preference
+  return (
+    <div>
```
This div is unnecessary.

jakemgilfix commented on components/Preferences.js:
```diff
@@ -0,0 +1,44 @@
+    }
+  ]
+  return (
+    <div>
```
Outer div doesn't do anything.

jakemgilfix commented on components/Preferences.js:
```diff
@@ -0,0 +1,44 @@
+  ]
+  return (
+    <div>
+      <main>
```
\<main\> should not be used here.

jakemgilfix commented on components/SaveChanges.js:
```diff
@@ -0,0 +1,17 @@
+    <div>
+      <main>
+        <div className="flex justify-center items-center">
+          <button className="bg-sunset-orange text-white w-[87.7%] h-12 rounded-[14px] mt-3">Save Changes</button>
```
Consider using a reusable button component, since these buttons should like consistent across the entire app.

jakemgilfix reviewed (changes requested):
Looks good, but there are some things that need to be addressed before it can be merged.

DanielEsquivelReynoso committed:
Update BackButton.js

implemented changes of removing the extra div and the main elements. also removed console.log

DanielEsquivelReynoso committed:
Update Preference.js

DanielEsquivelReynoso committed:
Update Preferences.js

DanielEsquivelReynoso committed:
Update ProfilePhoto.js

DanielEsquivelReynoso committed:
Update SaveChanges.js

DanielEsquivelReynoso committed:
Update SignOut.js

jakemgilfix committed:
Merge branch 'main' into settings
